### DISCUSSION TITLE
Show weekends with different colour

### DISCUSSION THREAD

**Discussion by oleg-andreyev** [community/community#109152] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Projects

### Body

As project manager it would be very benefical to see weekends
<img width="1560" alt="image" src="https://github.com/community/community/assets/1244112/b6ee4d9a-c10e-4a33-8407-0cd2fe305e57">

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by captn3m0** [comment]
I've been setting end dates of issues to weekends, only realizing the next week.

**Reply by captn3m0** [reply]
I resorted to writing a little custom CSS with Refined GitHub extension:

```
time[datetime="2024-07-27"], time[datetime="2024-07-28"], time[datetime="2024-08-02"], time[datetime="2024-08-03"], time[datetime="2024-08-09"] , time[datetime="2024-08-10"]     {
    background-color: lightcoral;
    color: white;
   border-radius: 3px;
}
```

![image](https://github.com/user-attachments/assets/fe3bb1fe-f334-4498-8006-935d2ae6d91f)
